src/goodbase/services/evg_service.py

```diff
@@ -103,11 +103,14 @@ def get_build_statuses_for_version(
         """
         if not evg_version.build_variants_status or len(evg_version.build_variants_status) == 0:
             return None
+        builds = evg_version.get_builds()
```

Comment:
Now with a single API call we get all builds from the version instead of making multiple API calls to get individual builds.